### PR TITLE
[ace]build_linux_pkgconfig

### LINKED ISSUE #21525: [ace] build failure
**Host Environment**
- OS: WSL (Ubuntu 20.04.3 LTS (GNU/Linux 4.4.0-19041-Microsoft x86_64)
- Compiler: g++ (Ubuntu 10.3.0-1ubuntu1~20.04) 10.3.0

**To Reproduce**
`./vcpkg install ace[tao,ssl,wchar,zlib]`

**Failure logs** 
```
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:155 (message):
    Command failed: make ssl=1 debug=1 optimize=0 -j9
    Working Directory: /mnt/d/vcpkg/buildtrees/ace/src/ACE-TAO-7-f61ead9ce9.clean/TAO
    See logs for more information:
      /mnt/d/vcpkg/buildtrees/ace/make-x64-linux-dbg-out.log
      /mnt/d/vcpkg/buildtrees/ace/make-x64-linux-dbg-err.log

Call Stack (most recent call first):
  ports/ace/portfile.cmake:347 (vcpkg_execute_build_process)
  scripts/ports.cmake:142 (include)
```
[mwc-x64-linux-out.log](https://github.com/microsoft/vcpkg/files/7566386/mwc-x64-linux-out.log)
[make-x64-linux-dbg-out.log](https://github.com/microsoft/vcpkg/files/7566383/make-x64-linux-dbg-out.log)
[make-x64-linux-dbg-err.log](https://github.com/microsoft/vcpkg/files/7566384/make-x64-linux-dbg-err.log)

**Additional context**
vcpkg HEAD: `cbc22a396c88d098247fe7f79f0913a952e1f77d`
When I downgrade to `8831ef24eee7e04009fe464097680512e46138a4` , I get an error inside `vcpkg_fixup_pkgconfig`.

### PR DESCRIPTION
**Pull request by pierrebizz**
**Describe the pull request**

Applies a patch to make the linux build succeed

- #### What does your PR fix?  
  Fixes #21525 

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  <linux, ...>, <Yes>

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `Yes`

### CODE REVIEW

File: ports/ace/ACETAO_dependencyFixed [hunk 1]
```diff
@@ -0,0 +1,26 @@
+diff --git a/TAO/orbsvcs/orbsvcs/TAO_CosProperty_Serv.pc.in b/TAO/orbsvcs/orbsvcs/TAO_CosProperty_Serv.pc.in
```
Comment (JackBoosY) [inline ports/ace/ACETAO_dependencyFixed:1]: This file name should be `fix-dependency.patch`.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — CHANGES_REQUESTED** [review]
```
-- Applying patch ACETAO_dependencyFixed
CMake Error at scripts/cmake/z_vcpkg_apply_patches.cmake:61 (message):
  Applying patch failed: Checking patch
  TAO/orbsvcs/orbsvcs/TAO_CosProperty_Serv.pc.in...

  error: TAO/orbsvcs/orbsvcs/TAO_CosProperty_Serv.pc.in: No such file or
  directory

  Checking patch TAO/tao/Monitor/TAO_Monitor.pc.in...

  error: TAO/tao/Monitor/TAO_Monitor.pc.in: No such file or directory
```
Please confirm this patch content.

**Comment by pierrebizz** [comment]
The problem was that my patch only applied to the tao target; would it be possible for the pipelines to test for all configurations ?

**Comment by jwillemsen** [comment]
https://github.com/DOCGroup/ACE_TAO/pull/1751

**Comment by JackBoosY** [comment]
Well, I think we should wait for the upstream merged your PR.

**Comment by pierrebizz** [comment]
Isn't it the idea of a patch not to have to wait for the upstream to fix it ?

**Comment by jwillemsen** [comment]
Different changes to fix this issue have been merged upstream, see https://github.com/DOCGroup/ACE_TAO/pull/1752. Your proposed patch as part of this PR has incorrect changes.

**Pull request closed without merging**